src/sentry/static/sentry/app/views/organizationIntegrations/docIntegrationDetailedView.tsx

```diff
@@ -67,6 +67,12 @@ class SentryAppDetailedView extends AbstractIntegrationDetailedView<
   }
 
   trackClick = () => {
+    if (window.location.href.includes('referrer=request_email')) {
+      this.trackIntegrationEvent({
+        eventKey: 'integrations.install_from_request',
```

Comment:
@ceorourke Why do we need a new event for this? We are tracking the `referrer` for all events so we can easily just look at the installation event when `custom_referrer= request_email`.